File: ArtOfIllusion/src/test/java/artofillusion/plugin/ImportDef.java
package artofillusion.plugin;

import com.thoughtworks.xstream.annotations.XStreamAlias;
import lombok.Data;
import com.thoughtworks.xstream.annotations.XStreamAsAttribute;

@XStreamAlias("import")
@Data
public class ImportDef {
    @XStreamAsAttribute private String name;
    @XStreamAsAttribute private String url;
}

File: xml-data/src/main/kotlin/artofillusion/plugin/ImportDef.kt
package artofillusion.plugin

import com.thoughtworks.xstream.annotations.XStreamAlias
import com.thoughtworks.xstream.annotations.XStreamAsAttribute

@XStreamAlias("import")
data class ImportDef(@XStreamAsAttribute val name: String? = null, @XStreamAsAttribute val url: String? = null)
